test/integration/features/basic.feature

```diff
@@ -3,14 +3,16 @@ Feature: Basic
   As a user
   I need to be able to bring up a test environment
 
-  Scenario: Basic test scenario
-    Given executing "minishift start --docker-env=FOO=BAR --docker-env=BAZ=BAT"
+  Scenario: Starting Minishift
+    Given Minishift should have state "Does Not Exist"
```

Comment:
We need a new step definition here or adjust the existing one to match also on "is in state". 
"Given" is a precondition which should be "Minishift [has|is in] state "Does Not Exist". "should have" makes no sense in this context.